types/sitTypes.ts

```diff
@@ -10,7 +10,7 @@ export type SitScheduleDay = {
 
 export type SitClass = {
     id: number,
-    activityId: string,
```

Comment:
Curiously enough, this is actually used as a number throughout the application, even though the type here said it should be a string. 🙃